frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -27,9 +27,31 @@ broker.id={{POD_INSTANCE_INDEX}}
 #     listeners = security_protocol://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
+{{^KAFKA_ENABLE_TLS}}
 listeners=PLAINTEXT://:{{KAFKA_BROKER_PORT}}
+{{/KAFKA_ENABLE_TLS}}
 
-# Hostname and port the broker will advertise to producers and consumers. If not set, 
+{{#KAFKA_ENABLE_TLS}}
+############################# TLS Settings #############################
```

Comment:
1. Technically its an easy solution but I don't see what additional security guarantees we'd get by creating a random keystore secret per task. Password would get injected to a container same way as JKS files. Also Java apps usually read a keystore password from a configuration file, which gets rendered by the `bootstrap` utility and gets written to a sandbox. If an attacker would get access to sanbox directory of any container (even not active) it would be possible to read the password and thus read a private key.